src/Gui/Document.cpp

```diff
@@ -786,7 +786,8 @@ void Document::slotChangedObject(const App::DocumentObject& Obj, const App::Prop
                     && d->_editingObject
                     && d->_editViewProviderParent 
                     && (Prop.isDerivedFrom(App::PropertyPlacement::getClassTypeId())
-                        || strstr(Prop.getName(),"Scale"))
+						// Issue ID 0004230 : getName() can return null in which case strstr() crashes
+						|| (Prop.getName() && strstr(Prop.getName(), "Scale")))

```

Comment:
You added Tabs to indent this code. Please don't add Tabs to code that is already indented with spaces.